src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -61,6 +60,7 @@ public class ScriptHost : JobHost, IScriptJobHost
         private readonly ILoggerFactory _loggerFactory = null;
         private readonly string _instanceId;
         private readonly IEnvironment _environment;
+        private static readonly int _processId = Process.GetCurrentProcess().Id;
```

Comment:
@safihamid -- here was another place we were calling `GetCurrentProcess()`, which I recall you identifying as expensive. We were calling it every time we started a host -- so moved it up here to cache it statically.

Reply:
Perfect! thanks @brettsam